build.gradle

```diff
@@ -156,7 +156,7 @@ spotless {
     format 'styling', {
         target createSpotlessTarget(['**/*.yaml', '**/*.json'])
 
-        prettier()
+//        prettier()
```

Comment:
How come you've commented out the `prettier()` method? presume this is simply pretty printing but regardless doesn't seem relevant to this PR

Reply:
Accidentally committed. Already rolled back.
How this could come? There was a version bumping change in check-style some time ago. Since then some linux users started seeing issues with importing and refreshing project in IntellijIdea but still works when executing build from console. I raised that issue but it seems like the author was busy with other tasks and wasn't going to switch back to his changes. I tried to check it in my free time and found that the issue comes from "prettier" and then stuck. Also found a few similar issues raised on the internet but without answers. So since then every time I have to comment it while working with code and then uncomment from time to time to run "./gradlew --no-daemon format --scan" command to format code and then comment it back. 
Here is what the issue looks like for me in IntellijIdea when I refresh gradle project with uncommented Prettier
![Selection_035](https://user-images.githubusercontent.com/4688354/199673750-c0312d10-9033-4a2e-bf49-4e5a2292d331.png)